app/models/solidus/affirm/checkout_payload.rb

```diff
@@ -1,4 +1,4 @@
-module Affirm
+module Solidus
```

Comment:
Dont define Solidus either.

Solidus is defined by Solidus. We don't own that namespace and thus we shouldn't be using it. 

Instead use something like `SolidusAffirm`

Also we'll want to fixup the past two commits to use the correct namespace instead of a separate commit.